namd/tests/library/multiple_walker_abf/run_mw_tests.sh

```diff
@@ -11,16 +11,25 @@ if (( $# != 1 )); then
 fi
 NAMD_EXEC=$1
 
-#NAMD_EXEC=$HOME/Software/NAMD_2.10b1_Source_colvars/Linux-x86_64-g++/namd2
+# Runner for the netlrts version
+RUNNER="${HOME}/progs/namd/Linux-x86_64-g++.netlrts/charmrun ++local +p4"
+
+# Runner for the MPI version
+#RUNNER="mpirun -n 4"
```

Comment:
Is there any chance that this could be run as part of the CI, using the MPI build of NAMD?

Reply:
Yes, that would be an excellent idea.